deploy/helm/flightctl/charts/ui/values.yaml

```diff
@@ -41,4 +41,7 @@ api:
   url: https://flightctl-api:3443/
   insecureSkipTlsVerify: false
   caCert: ""
-
+## @param cliArtifacts.enabled Set to true if Cli artifacts are enabled in the UI
+# TODO should it be part of global??
```

Comment:
I believe it would be better to move whether the cliArtifacts are enabled to the global scope.
See https://github.com/flightctl/flightctl/commit/03561147a21f02f3c65d1ab06ea94f65b7553f43

Reply:
Discussed with @rawagner , if we do this, it should apply to all other components and in a separate PR.
Keeping this PR with the current approach.